frontend/smart-kickers-game/src/components/Game/GameResults/GameResults.js

```diff
@@ -15,7 +15,7 @@ function GameResults({ blueScore, whiteScore }) {
 
   return (
     <div>
-      <div className="game-result-container">
+      <div className="game-result-container" hidden={!isVisible}>
```

Comment:
I would say that using `hidden` property is kind of a bad practice. It's better to use a conditional rendering in JSX. The comment is about all the places where you did that.

Reply:
Btw, the styles you've added won't be needed anymore if you changed that, so don't forget to remove them.

Reply:
Done as you suggested.